server/app/services/export/JsonExporter.java

```diff
@@ -206,4 +228,13 @@ private DocumentContext makeEmptyJsonArray() {
   private DocumentContext makeEmptyJsonObject() {
     return JsonPathProvider.getJsonPath().parse("{}");
   }
+
+  public String getFormattedPhoneNumber(String phoneNumberValue, String countryCode) {
```

Comment:
**suggestion (blocking):** methods show have the most restrictive visibility setting they can. Make this method `private` or if you're testing directly make it package-private (remove the visibility keyword) and add the `@VisibleForTesting` annotation

Reply:
Done!